data/metadata.json

```diff
@@ -613,6 +613,9 @@
     "hack-toolbox/lockscreen/open/lock.com.endlessm.Hackdex_chapter_one.1": {
         "sound-file": "toolbox/Lockscreen_hackdex_ch1_open.webm"
     },
+    "hack-toolbox/lockscreen/open/lock.com.endlessm.Hackdex_chapter_two.1": {
```

Comment:
I see this matches `play(`hack-toolbox/lockscreen/open/${this._lock}`)` in the toolbox, :+1: